This patch comes from in-house and this is Solaris specific patch to use
the GNU version of soelim.

--- groff-1.22.2/contrib/groffer/perl/func.pl.orig	2013-11-20 13:51:20.387873317 -0800
+++ groff-1.22.2/contrib/groffer/perl/func.pl	2013-11-20 13:52:29.569174110 -0800
@@ -271,7 +271,7 @@
 
   my $soelim_r = '';
   $soelim_r = '-r'
-    if ! system("echo -n '' | soelim -r 2>$main::Dev_Null >$main::Dev_Null");
+    if ! system("echo -n '' | /usr/gnu/bin/soelim -r 2>$main::Dev_Null >$main::Dev_Null");
 
   ##########
   # to_tmp (<filename>)
@@ -410,7 +410,7 @@
 
       unlink $tmp_tmp if -e $tmp_tmp;
       rename $tmp_file, $tmp_tmp;
-      system("soelim -I$dir $soelim_r $tmp_tmp >$tmp_file");
+      system("/usr/gnu/bin/soelim -I$dir $soelim_r $tmp_tmp >$tmp_file");
       unlink $tmp_tmp if -e $tmp_tmp;
 
     } else {			# $Filespec_Is_Man is empty
@@ -421,9 +421,9 @@
       }
       close $fh_tmp;
       if ($dir) {
-	system("soelim -I$dir $soelim_r $tmp_tmp >$tmp_file");
+	system("/usr/gnu/bin/soelim -I$dir $soelim_r $tmp_tmp >$tmp_file");
       } else {
-	system("soelim $soelim_r $tmp_tmp >$tmp_file");
+	system("/usr/gnu/bin/soelim $soelim_r $tmp_tmp >$tmp_file");
       }
       unlink $tmp_tmp;
     }				# if ($Filespec_Is_Man)
